sandbox/Makefile

```diff
@@ -1,9 +1,21 @@
-CFLAGS=-I../include -lGL -lm -lX11
+CFLAGS=-I../include
+
+ifneq (,$(findstring win32,$(MAKECMDGOALS)))
+CFLAGS+= -lm -lopengl32
+else
+CFLAGS+= -lGL -lm -lX11
+endif
 
 .PHONY: unix-glx
 
 unix-glx: main.c ../build/unix-glx/libbreadbox.a
 	$(CC) $(CFLAGS) -o ../build/unix-glx/sandbox $^
 
+win32-wgl: main.c ../build/win32-wgl/libbreadbox.a
```

Comment:
Strange. I thought static libraries ended in .lib on Windows. Not a bit deal, but I'm still curious.